glide.lock

```diff
@@ -1,21 +1,54 @@
 hash: 7d1e85d070980bf11cab0f4dc869c2af64ecfaf41b1ccb52eaa5db06184b23ba
-updated: 2017-05-03T14:11:01.313181441-07:00
+updated: 2017-06-02T11:28:58.980126559-07:00
```

Comment:
you're absolutely right. I normally do that. I should've just done that in this case. Will do in the future.